docs/articles/viewing-epoch-metrics-webui.rst

```diff
@@ -0,0 +1,167 @@
+.. _coreapi-epoch-metrics-howto:
+
+##############################################
+ How to View Epoch-Based Metrics in the WebUI
+##############################################
+
+.. meta::
+   :description: Learn how to view epoch-based metric data in the WebUI by reporting metrics to the Core API.
+   :keywords: CoreAPI, WebUI, epoch-based, metrics, metric data
+
+You can view epoch-based metric data in the WebUI when you report metrics to the Determined master
```

Comment:
| You can view epoch-based metric data in the WebUI when you report metrics

I would change this sentence to specifically say when you report an `epoch` metric, or by reporting `epoch-based metrics`. This sentence reads as if just by reporting metrics you can view them based on epochs which is not the case. However, the rest of the article is very clear about having to report epoch.

Reply:
resolved